draft-irtf-cfrg-hpke.md

```diff
@@ -343,7 +334,7 @@ HPKE variants rely on the following primitives:
     returning plaintext message `pt`. This function can raise an
     `OpenError` or `NonceOverflowError` upon failure.
   - `Nk`: The length in bytes of a key for this algorithm
-  - `Nn`: The length in bytes of a nonce for this algorithm
+  - `Nn`: The length in bytes of a nonce (and initialization vector) for this algorithm
```

Comment:
I agree that we only need one length.

I am less sure about removing the word "nonce" completely, because the AEADs we use call it nonce. Citing from the original feedback #152, it was suggested to align with TLS: “in TLS 1.3 for example, the "nonce" is the initialization vector XORed with a sequence number.”

Reply:
@bifurcation I converged on `base_nonce` and `nonce` for the KDF output and AEAD input, respectively.